src/helpers/approvals-satisfied.ts

```diff
@@ -50,7 +50,7 @@ export const approvalsSatisfied = async ({ teams, number_of_reviewers = '1', pul
     });
     const codeOwnerLogins = teamsAndLoginsLists.flat().map(({ login }) => login);
 
-    const numberOfCollectiveApprovalsAcrossTeams = approverLogins.filter(login => codeOwnerLogins.includes(login)).length;
+    const numberOfCollectiveApprovalsAcrossTeams = approverLogins.filter(login => login && codeOwnerLogins.includes(login)).length;
```

Comment:
I think this would only matter if GitHub sent us an empty string login, but this is fine

Reply:
yeah it was just to get the build to pass